src/TagLib/Matroska/Tag.cs

```diff
@@ -384,6 +384,7 @@ public string[] Get(string key, string subkey = null, bool recu = true)
 						{
 							str = null;
 						}
+						else
```

Comment:
Could you put this else block in `{` `}`?

Reply:
the `else` block conditions the next `if`. IMHO, a better cosmetic fix would be the put the `if` on the same line to get an `else if` construct.

Reply:
I'm a fan of whatever makes the code more readable. If it is an `else if` then agreed they need to be on the same line :)